superset-frontend/src/explore/components/controls/DateFilterControl.jsx

```diff
@@ -82,8 +82,8 @@ const DEFAULT_SINCE = moment()
 const DEFAULT_UNTIL = moment().utc().startOf('day').format(MOMENT_FORMAT);
 const SEPARATOR = ' : ';
 const FREEFORM_TOOLTIP = t(
-  'Superset supports smart date parsing. Strings like `last sunday` or ' +
-    '`last october` can be used.',
+  'Superset supports smart date parsing. Strings like `3 weeks ago`, `last sunday` or ' +
```

Comment:
oh, one thing I missed if we do stick with the current approach. Can you add the Oxford Comma before `or` here? Thanks!